doc/Makefile

```diff
@@ -41,7 +41,9 @@ ifeq ($(wildcard ../info),../info)
 endif
 
 zip:
+ifneq "$(wildcard ../html)" ""
 	@cd .. && zip cc65 html/*.*
+endif
```

Comment:
Yes. Without this, if the html directory was not created, the zip target would fail. I felt it'd be consistent that if we allow the build to pass without building docs then we shouldn't demand them to be always present here.